.flake8

```diff
@@ -0,0 +1,21 @@
+# Right now we use Flake8 only for a few static checks focusing on runtime
+# safety and correctness.  We don't use it for style checks.
+
+[flake8]
+select =
+    # syntax errors
+    E9,
+
+    # all pyflakes correctness issues
+    F,
+
+extend-ignore =
+    # allow f-strings without any placeholders
+    F541,
+
+    # allow unused variables
+    F841,
+
+per-file-ignores =
+    # allow unused imports for re-export
+    augur/io/__init__.py:F401
```

Comment:
This works locally but [doesn't in CI](https://github.com/nextstrain/augur/actions/runs/5954918209/job/16152462942#step:8:332).

I'll probably resort to inline exceptions.